tests/integration/docusaurus/deployment_patterns/aws_glue_deployment_patterns.py

```diff
@@ -12,6 +12,11 @@
 )
 from great_expectations.util import get_context
 
+import warnings
+
+# needed because GlueContext(sc) function emits the following FutureWarning: Deprecated in 3.0.0. Use SparkSession.builder.getOrCreate() instead.
+warnings.filterwarnings("ignore", category=FutureWarning)
```

Comment:
This is a very broad filter.
I was going to suggest using a context manager to just filter the spot where this is being emitted.
But this also isn't core gx code so NBD.

Reply:
Follow up PR is #8466